Watchman.DomainModel/Responses/ResponsesMessageService.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Devscord.DiscordFramework.Commons.Extensions;
+using Devscord.DiscordFramework.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Resources = Devscord.DiscordFramework.Framework.Commands.Responses.Resources;
+
+namespace Watchman.DomainModel.Responses
+{
+    public class ResponsesMessageService
+    {
+        private ResponsesDatabase _responsesDatabase;
+        private MessagesService _messagesService;
+        private const int MAX_NUMBER_OF_MESSAGE_FIELDS = 20;
+
+        public ResponsesMessageService(ResponsesDatabase responsesDatabase, MessagesService messagesService)
+        {
+            _responsesDatabase = responsesDatabase;
+            _messagesService = messagesService;
+        }
+
+        public Task PrintResponses(string commandArgument)
+        {
+            if (commandArgument == "default")
+            {
+                PrintDefaultResponses();
+            }
+
+            if (commandArgument == "custom")
+            {
+                PrintCustomResponses();
+            }
+
+            if (commandArgument == "all")
+            {
+                PrintAllResponses();
+            }
+
+            return Task.CompletedTask;
+        }
+
+        private void PrintDefaultResponses()
+        {
+            var messagesAboutDefaultResponses = GetMessagesAboutDefaultResponses();
+            for (int i = 0; i < messagesAboutDefaultResponses.Count; ++i)
+            {
+                if (i == 0)
+                {
+                    _messagesService.SendEmbedMessage("Domyślne responses:", "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/", messagesAboutDefaultResponses[0]);
+                }
+
+                else
+                {
+                    _messagesService.SendEmbedMessage(title: null, description: null, messagesAboutDefaultResponses[i]);
+                }
+            }
+        }
+
+        private void PrintCustomResponses()
+        {
+            var messagesAboutCustomResponses = GetMessagesAboutCustomResponses();
+            for (int i = 0; i < messagesAboutCustomResponses.Count; ++i)
+            {
+                if (i == 0)
+                {
+                    _messagesService.SendEmbedMessage("Nadpisane responses:", description: null, messagesAboutCustomResponses[0]);
+                }
```

Comment:
Może lepiej to przed pętlę wyciągnąć, wtedy nie trzeba będzie sprawdzać czy i == 0
samą pętlę wtedy zacznij od i = 1